Link Fig1 png by url

cbethell opened the pull request:
### Purpose

Following the plan to get publication figures ready, I am adding the url to png for figure 1, the OpenPBTA overview figure.

### Issue
This is a part of issue [#571](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/571#issue-572964297) in the OpenPBTA-analysis repository.

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
Is there anything else that needs to be done at this time? 

**Note:** This figure is not yet complete as the three remaining panels still need to be generated (the panels that relate to Github contribution statistics, features of the project, and the kind of assays in the project). 

I therefore left the figure sketch for reference to what the remaining aspects of the panel will look like.

#### Is the pull request ready for review?
Yes. 

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

None of these changes were made.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

jaclyn-taroni commented on content/04.results.md:
```diff
@@ -6,7 +6,9 @@ Results section stub.
 This section will introduce the dataset (e.g., the histologies represented and what data types are included; Figure {@fig:openpbta-overview}A-B) and the process for contributing analytical code and to the manuscript (Figure {@fig:openpbta-overview}C-D).
 
-![An overview of the OpenPBTA project](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview}
+![An overview of the OpenPBTA project sketch](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview-sketch}
```
Can you remove this please? I would also remove the PNG of the sketch from the repository.

jaclyn-taroni commented on content/04.results.md:
```diff
@@ -6,7 +6,9 @@ Results section stub.
-![An overview of the OpenPBTA project](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview}
+![An overview of the OpenPBTA project sketch](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview-sketch}
+
+![An overview of the OpenPBTA project](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/fig1-openpbta-distribution.png){#fig:openpbta-overview}
```
Can you add a figure legend for panel A? See @jashapiro's example on #91.

cbethell commented:
@jaclyn-taroni, I added what I believe to be a figure legend for panel A in the [last commit](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/90/commits/9f548e23cf7750c017f2ecd1a17d0a57dac7b440).

Would you mind checking to ensure that this is the case.
Also, does the description that I added there appear to be suffice?

jaclyn-taroni commented on content/04.results.md:
```diff
@@ -6,7 +6,9 @@ Results section stub.
 
-![An overview of the OpenPBTA project](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview}
+![An overview of the OpenPBTA project.
+A) The distribution of samples across short histologies and integrated diagnoses.
```
I think this could use a bit more detail at least while we are writing this up. When you say `samples` what do you mean? Is it biospecimens? Is it `sample_id`? Participants?
